scripts/build.js

```diff
@@ -1,115 +1,119 @@
 // Do this as the first thing so that any code reading it knows the right env.
-process.env.NODE_ENV = 'production';
-
-var chalk = require('chalk');
-var fs = require('fs');
-var path = require('path');
-var filesize = require('filesize');
-var gzipSize = require('gzip-size').sync;
-var rimrafSync = require('rimraf').sync;
-var webpack = require('webpack');
-var config = require('../config/webpack.config.prod');
-var paths = require('../config/paths');
-var checkRequiredFiles = require('./utils/checkRequiredFiles');
-var recursive = require('recursive-readdir');
-var stripAnsi = require('strip-ansi');
-var formatMessage = require('./utils/utils').formatMessage;
-var isLikelyASyntaxError = require('./utils/utils').isLikelyASyntaxError;
-
-checkRequiredFiles();
+import chalk from 'chalk'
+import fs from 'fs'
+import path from 'path'
+import filesize from 'filesize'
+import gzipSize from 'gzip-size'
```

Comment:
I think you removed `gzip-size` from devDependencies -- right now this is getting picked up from `react-dev-utils`.  I'd rather see explicit dependencies for things our code imports.